packet/flatcar-linux/kubernetes/variables.tf

```diff
@@ -127,3 +127,52 @@ variable "enable_aggregation" {
   type        = "string"
   default     = "false"
 }
+
+variable "worker_user_data" {
+  description = "User data of the worker"
+  type        = "string"
+  default     = ""
+}
+
+variable "auth_token" {
+  description = "Packet auth token"
+  default     = ""
+  type        = "string"
+}
+
+variable "worker_type" {
+  description = "Packet instance type for workers"
+  default     = "baremetal_0"
+  type        = "string"
+}
+
+variable "worker_os_channel" {
+  description = "Flatcar Linux channel to install from (stable, beta, alpha, edge) on workers"
+  default     = "stable"
+  type        = "string"
+}
+
+variable "enable_autoscaler" {
```

Comment:
Shouldn't that be named `worker_enable_autoscaler`? What if we want to have autoscaling enabled for controlplane later on?